nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/resources/OSGI-INF/copy-schema-test-contrib.xml

```diff
@@ -2,5 +2,13 @@
 <component name="org.nuxeo.ecm.automation.core.operations.document.copyschema.test">
   <extension target="org.nuxeo.ecm.core.schema.TypeService"
              point="doctype">
+    <doctype name="CommentRoot" extends="Folder" isSpecial="true">
```

Comment:
Can be removed too isn't it?

Reply:
yes this is only meant for the `CopyDocumentTest` about to disappear